quickstarts/dotnet/blazor-server/config.yml

```diff
@@ -0,0 +1,48 @@
+slug: blazor-server
+
+description: |
+  ## Comprehensive monitoring quickstart for Blazor Server
+  Microsoft developed Blazor Server to build web UIs with C#, HTML, and CSS. As a feature of ASP.NET, our Blazor Server integration with the .NET agent lets you monitor your apps with our quickstart dashboards right out of the box.
```

Comment:
"Our Blazor Server integration with the .NET APM agent lets you monitor your apps with our dashboards right out of the box."